collectors/UcsmCRCFaultCollector.py

```diff
@@ -13,6 +17,7 @@ def get_metrics(self):
                                          }
 
     def collect_metrics(self, server, handle):
+        logger.info("Collecting Metrics ")
```

Comment:
Could this be debug?

Reply:
Added this to get the info about whats going on right now without debug mode.

Reply:
This tends to stay in there forever and will generate endless logs in default mode. Run debug if you want to see more output. If you see your metrics in prometheus, you will know, that it is working. Or the other way round, if the code is not failing, it will work too ;). This adds no real benefit imho.

Reply:
Yes aggreed. Changed to Debug :)